src/test/java/teammates/test/cases/ui/InstructorFeedbackQuestionEditActionTest.java

```diff
@@ -1400,7 +1400,7 @@ public void testExecuteAndPostProcessResponseRate() throws Exception {
         fs = fsLogic.getFeedbackSession(fs.getFeedbackSessionName(), fs.getCourseId());
         details = fsLogic.getFeedbackSessionDetails(fs);
         assertEquals(numStudentRespondents, details.stats.submittedTotal);
-        assertEquals(totalStudents + 1, details.stats.expectedTotal);
```

Comment:
Originally only `"instructor1@course1.tmt"` was an expected instructor respondent for that session (thats where the `+ 1` comes from). After adding a sample question to `"First feedback session"` with `"instructor2@course1.tmt"` as a giver in its custom feedback paths, now both `"instructor1@course1.tmt"` and `"instructor2@course1.tmt"` are also expected instructor respondents in that session, hence the `+ 2`.